config/environments/development.rb

```diff
@@ -45,19 +45,13 @@
   # number of complex assets.
   config.assets.debug = true
 
+  config.action_mailer.delivery_method = :postmark
+  config.action_mailer.postmark_settings = { :api_token => ENV['POSTMARK_TOKEN'] }
+
+  config.action_mailer.raise_delivery_errors = true
   config.action_mailer.default_url_options = { :host => 'localhost:3000' }
   config.action_mailer.perform_deliveries = true
-  config.action_mailer.delivery_method = :smtp
   config.action_mailer.default :charset => "utf-8"  
-  config.action_mailer.smtp_settings =  {
-    :address  => "smtp.postmarkapp.com",
```

Comment:
Was postmark SMTP config already in place, so this is just a switch to using their API?

Reply:
We were using Postmark for local development builds. I switched to the Postmark API partly because they strongly encourage it and partly because it seems to be better at logging errors.